app/util/data_preparation/confluence/api.py

```diff
@@ -5,15 +5,10 @@
 import requests
 import yaml
 
+from util.conf import CONFLUENCE_SETTINGS
 
-def admin_credentials():
-    # TODO extract paths to project_paths
-    with open(Path(__file__).parents[3] / "confluence.yml", 'r') as file:
-        jira_yaml = yaml.load(file, Loader=yaml.FullLoader)
-        return jira_yaml['settings']['env']['admin_login'], jira_yaml['settings']['env']['admin_password']
 
-
-USER, PASSWORD = admin_credentials()
+USER, PASSWORD = CONFLUENCE_SETTINGS.admin_login, CONFLUENCE_SETTINGS.admin_password
```

Comment:
try to rewrite this class do not import util.conf but take user and password as input parameters in the constructor, see ApiJira for reference

Reply:
Good idea, I'll rewrite